python/backup.py

```diff
@@ -0,0 +1,25 @@
+import sqlite3
+import os
+import sys
+
+if len(sys.argv) != 2:
```

Comment:
Nit: Can you use argparse instead?
As well as can you put this under a main.
Example of both here: https://github.com/Hivemapper/sensor_fusion/blob/main/sensor_fusion_service/main.py#L269